00-HTML-CSS-basics/Ex01_V_VI.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title> Trying more things </title>
+	</head>
+
+	<body>
+		<header>
+			<h1> Excercises V and VI </h1> 
+		</header>
+		<img src="http://orig00.deviantart.net/bb48/f/2014/088/8/9/jack_frost_persona__by_flaremor-d7c4r75.png" alt="This is Jack Frost from the SMT Series" width="20%"> <br>
+		<p> This is Jack Frost from the SMT Series <br> </p>
+		<video width="50%" preload="none" controls>
+			<source src="Media/Jack_Frost_Converted.mp4" type="video/mp4">
```

Comment:
We should be aware of the cases when a feature is not supported by the browser. An additional message that say "Your browser does not support `source` tag for example". Remember the advice for the next time.